examples/armv4t/gdb/mod.rs

```diff
@@ -36,6 +36,14 @@ fn cpu_reg_id(id: ArmCoreRegId) -> Option<u8> {
     }
 }
 
+pub fn copy_range_to_buf(data: &[u8], offset: u64, length: usize, buf: &mut [u8]) -> usize {
```

Comment:
please add a (brief) comment documenting the purpose and behavior of this function (notably, its behavior with EOF conditions)